content/oci/oci.go

```diff
@@ -43,9 +43,9 @@ import (
 // Reference: https://github.com/opencontainers/image-spec/blob/v1.1.0-rc5/image-layout.md
 type Store struct {
 	// AutoSaveIndex controls if the OCI store will automatically save the index
-	// file on each Tag() call.
-	//   - If AutoSaveIndex is set to true, the OCI store will automatically call
-	//     this method on each Tag() call.
+	// file when needed.
+	//   - If AutoSaveIndex is set to true, the OCI store will automatically save the
+	//     changes to `index.json` on Tag() and Delete() calls, and when pushing a manifest.
```

Comment:
`on Tag() and Delete() calls, and when pushing a manifest` --> `on tag changes of Push(), Tag(), Delete() calls`.

Reply:
"tag changes" might be misleading, as we also save untagged manifest in `index.json`. I think the current description is OK.🤔

Reply:
@Wwwsylvia How about delete a blob? That call won't trigger `Delete()`

Reply:
Blobs can also be tagged. If we delete a tagged blob, the `index.json` should also be updated.

Reply:
Refined the wording